### PR TITLE
Implement alternative AE/BE keywords spelling

### PR DESCRIPTION
**Pull request by aradi**
Node names, where American and British English versions differ, are converted internally to BE, allowing for both versions in the input independent of the InputVersion/ParserVersion settings.

### CODE REVIEW

**Comment by aradi** [comment]
I'll change further keywords, don't merge it yet...

**Comment by aradi** [comment]
@bhourahine  I've have split the commit into two: the `destroyNodeList()` fix and the AE spelling. I've updated the manual as well and implemented AE/BE alternatives for each keyword I could identify as dual-spelled. Should be ready for review now.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -18,6 +18,9 @@ Added
 
 - Spin-orbit coupling for xTB
 
+- Alternative American English spelling for various input keywords
```
Comment (bhourahine) [inline CHANGELOG.rst:21]:
```suggestion
- Dual American and British English spelling for various input keywords
```

**Review by bhourahine — APPROVED** [review]
Ready to merge, but see suggestion for the change log

**Pull request merged**